src/components/EditCoursePage/__snapshots__/AdditionalMetadataFields.test.jsx.snap

```diff
@@ -79,6 +79,22 @@ exports[`AdditionalMetadata Fields Display all fields 1`] = `
       name="additional_metadata.organic_url"
       required={true}
     />
+    <Field
+      component={[Function]}
+      disabled={false}
+      label={
+        <FieldLabel
+          className=""
+          extraText=""
+          helpText=""
+          id="variant_id.label"
+          optional={false}
+          text="Course Variant Id"
+        />
+      }
+      name="additional_metadata.variant_id"
+      required={false}
```

Comment:
note here the filed is no longer required

Reply:
Same as before. Required is by default a false field.